src/build/build-workflow.ts

```diff
@@ -112,6 +112,14 @@ export interface BuildWorkflowOptions {
   readonly permissions?: JobPermissions;
 }
 
+/**
+ * Options for WorkflowOptions.runsOn
+ */
+export interface GroupRunnerOptions {
+  readonly group: string;
+  readonly labels: string[];
```

Comment:
I believe `labels` is optional. 
```suggestion
  readonly labels?: string[];
```

Source: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-groups-to-control-where-jobs-are-run

Reply:
Good note! Fixed this in https://github.com/projen/projen/pull/2817/commits/b2ae37e270d5974c1293ab5b6e06f3954897a9df